sdk/python/packages/flet-core/src/flet_core/column.py

```diff
@@ -169,6 +171,15 @@ def clean(self):
     async def clean_async(self):
         self.clean()
 
+    def add(self, *controls: Control) -> None:
+        self.__add(self.__controls, *controls)
+
+    def insert(self, at: int, *controls: Control) -> None:
+        self.__insert(self.__controls, at, *controls)
+
+    def remove(self, *controls: Control) -> None:
+        self.__remove(self.__controls, *controls)
+
     # tight
```

Comment:
Could we remove all occurrences of `add`, `insert` and `remove` (and underlying logic) from the PR please?
1) We would have auto-update related feature in a separate PR and 2) I've been recently thinking a lot about auto-update feature, which is of course will be great (and maybe "a must") addition to Flet, but came out with a different approach. Those `add/insert/remove` was added in "experimental" PR by @ndonkoHenri and, perhaps, won't be needed at all in a new approach.

I think we could close https://github.com/flet-dev/flet/pull/3053 as well.

Do we have an issue to discuss auto-update?

Reply:
> Could we remove all occurrences of `add`, `insert` and `remove` (and underlying logic) from the PR please?
> 
> 1. We would have auto-update related feature in a separate PR and 2) I've been recently thinking a lot about auto-update feature, which is of course will be great (and maybe "a must") addition to Flet, but came out with a different approach. Those `add/insert/remove` was added in "experimental" PR by @ndonkoHenri and, perhaps, won't be needed at all in a new approach.
> 
> I think we could close #3053 as well.
> 
> Do we have an issue to discuss auto-update?

I want to implement this feature now because it's logical (Page have `add()`, `insert()` and `remove()`. If i have to remove this methods, i have to remove `__add__()`, `__radd__()`, `__iadd__()` and other magic methods. Also i already did `add/insert/remove` methods in [#281](https://github.com/flet-dev/website/pull/281). I don't want to re-do this and that PR.

Why i and other contributors can't do `add/insert/remove` methods?

Reply:
I appreciate your time working on this PR, but auto-update feature won't be working like "change collection, update control/page" - thus those methods are unnecessary. Without `self.update` these methods don't add any value (`self.controls.append(something)` vs `self.add(something)`) and just make API more complex.

Reply:
> I appreciate your time working on this PR, but auto-update feature won't be working like "change collection, update control/page" - thus those methods are unnecessary. Without `self.update` these methods don't add any value (`self.controls.append(something)` vs `self.add(something)`) and just make API more complex.

Thank you for feedback, but for me these methods is very comfortable. It's possible to make these methods and not making API more complex? Also how to make these methods less unnecessary?

Reply:
Let's try avoiding to do mega-PRs including multiple features (I do my best to hold myself putting everything into one PR as well :) If you are able to break your PR(s) into smaller PRs then we could discuss them separately.

I'm not saying these methods are not usable or unnecessary I just don't want to include them into core API now to keep things simple. Not everything could be included into core. Hope you understand. If you need those methods you can create a separate package "patching" Flet classes and put it on pypi.org.

For better communication I'm going to setup a public channel for Flet developers contributing or willing to contribute to Flet core. There we could discuss whether this or that functionality is desirable, how to implement this. Also, we are going to have periodic video hang-outs there.

Reply:
> Let's try avoiding to make mega-PRs including multiple features (I do my best to hold myself putting everything into one PR as well :) If you are able to break your PR(s) into smaller PRs then we could discuss them separately. It's easier for developers and maintainers.
> 
> I'm not saying these methods are not usable or unnecessary I just don't want to include them into core API now to keep things simple. Not everything could be included into core. Hope you understand. If you need those methods you can create a separate package "patching" Flet classes and put it on pypi.org.
> 
> For better communication I'm going to setup a public channel for Flet developers contributing or willing to contribute to Flet core. There we could discuss whether this or that functionality is desirable, how to implement this. Also, we are going to have periodic video hang-outs there.

Ok, i'll remove this methods

Reply:
#3329